diff --git a/node_modules/@walletconnect/iso-crypto/dist/cjs/index.js b/node_modules/@walletconnect/iso-crypto/dist/cjs/index.js
index 8d4a064..8cd6a9d 100644
--- a/node_modules/@walletconnect/iso-crypto/dist/cjs/index.js
+++ b/node_modules/@walletconnect/iso-crypto/dist/cjs/index.js
@@ -7,7 +7,7 @@ const utils_1 = require("@walletconnect/utils");
 function generateKey(length) {
     return tslib_1.__awaiter(this, void 0, void 0, function* () {
         const _length = (length || 256) / 8;
-        const bytes = isoCrypto.randomBytes(_length);
+        const bytes = Buffer.from(yield require('crypto').randomBytes(_length), 'base64')
         const result = utils_1.convertBufferToArrayBuffer(encUtils.arrayToBuffer(bytes));
         return result;
     });
