diff --git a/node_modules/react-native-text-size/android/build.gradle b/node_modules/react-native-text-size/android/build.gradle
index 2617bd7..1e74d6a 100644
--- a/node_modules/react-native-text-size/android/build.gradle
+++ b/node_modules/react-native-text-size/android/build.gradle
@@ -8,8 +8,8 @@ def safeExtGet(prop, fallback) {
     return rootProject.ext.hasProperty(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-def _buildToolsVersion  = safeExtGet('buildToolsVersion', '28.0.3')
-def _compileSdkVersion  = safeExtGet('compileSdkVersion', 28)
+def _buildToolsVersion  = safeExtGet('buildToolsVersion', '34.0.0')
+def _compileSdkVersion  = safeExtGet('compileSdkVersion', 34)
 def _targetSdkVersion   = safeExtGet('targetSdkVersion', 28)
 def _minSdkVersion      = safeExtGet('minSdkVersion', 16)
 
